schema-supplement.rst

```diff
@@ -23,13 +23,18 @@ By definition, none of the following attributes are required.
 :attribute:`default_license`
 ----------------------------
 
+:Type: :type:`string`
+:Applies To: :object:`package`
+
 Specifies the `license`_ that is assumed to apply to a component,
 if none is otherwise specified.
 This is convenient for packages
 that wish their `license`_ to reflect portions of the package
 that are not reflected by a component (such as data files)
 when most or all of the compiled artifacts use the same license.
 
+This field shall be a valid |SPDX|_ `License Expression`_.
```

Comment:
Ack, apologies for not noticing sooner, but we don't actually use "field" *anywhere* in the extant spec.

Let's copy the later text instead:
```suggestion
The value shall be a well formed
|SPDX|_ `License Expression`_ .
```